cypress/support/commands.js

```diff
@@ -64,4 +87,84 @@ Cypress.Commands.add('createFleet', () => {
+    cy.get('.pf-v5-c-table__action > .pf-v5-c-menu-toggle').click()
+    cy.get('[data-ouia-component-id="OUIA-Generated-DropdownItem-8"] > .pf-v5-c-menu__item > .pf-v5-c-menu__item-main > .pf-v5-c-menu__item-text').click()
+    cy.get(':nth-child(2) > .pf-v5-c-form__group-label > .pf-v5-c-form__label > .pf-v5-c-form__label-text').should('contain', 'Fleet name')
+    cy.get('.pf-v5-c-form__group-control > .pf-v5-c-label-group > .pf-v5-c-label-group__main > .pf-v5-c-label-group__list > .pf-v5-c-label-group__list-item > .pf-v5-c-button').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-title').should('contain', 'test-fleet')
+})
+Cypress.Commands.add('deleteFleet', () => {
+    cy.get('#nav-toggle').click()
+    cy.contains('Edge Management').click()
+    cy.contains('Fleets').click()
+    cy.get('[data-label="Name"]').contains('test-fleet')
+    cy.get('.pf-v5-c-table__tbody > .pf-v5-c-table__tr > .pf-v5-c-table__check > label > input').click()
+        cy.get('#pf-random-id-63 > :nth-child(1) > .pf-v5-c-toolbar__content-section > :nth-child(3) > .pf-v5-c-button').click()
+        cy.get('.pf-m-danger').should('be.visible')
+    cy.get('.pf-m-danger').click()
+})
+Cypress.Commands.add('importFleet', () => {
+    cy.get('#nav-toggle').click()
+    cy.contains('Edge Management').click()      
+    cy.contains('Fleets').click()
+    cy.get(':nth-child(2) > .pf-v5-l-split > :nth-child(2) > .pf-v5-c-button').should('exist')
+    cy.get(':nth-child(2) > .pf-v5-l-split > :nth-child(2) > .pf-v5-c-button').should('be.visible')
+    cy.get(':nth-child(2) > .pf-v5-l-split > :nth-child(2) > .pf-v5-c-button').click()
+    cy.get('#rich-validation-field-name').type('test-fleet')
+    cy.get('#textfield-url').type('https://github.com/flightctl/flightctl-demos')
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('#rich-validation-field-resourceSyncs\[0\]\.name').type('test-resource')
+    cy.get('#textfield-resourceSyncs\[0\]\.targetRevision').type('main')
+    cy.get('#textfield-resourceSyncs\[0\]\.path').type('/demos/basic-nginx-demo/deployment/fleet.yaml')
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-wizard__footer > .pf-m-primary').click()
+    cy.get('.pf-v5-c-alert__title').should('contain', 'Importing fleets')
+    cy.get('.fctl-resource-link__text', { timeout: 50000 }).should('contain', 'basic-nginx-fleet')
+})
+Cypress.Commands.add('downloadFile', (os = `${Cypress.env('platform')}`, arch = `${Cypress.env('arch')}`) => {
+    cy.get('[data-test="help-dropdown-toggle"]').click()
+    cy.contains('Command Line Tools').should('be.visible')
+    cy.contains('Command Line Tools').click()
+    cy.contains(`Download flightctl for ${os} for ${arch}`).downloadFile()
```

Comment:
I'm confused about this command.

Is  this line 150 invoking a separate "downloadFile" [plugin](https://www.npmjs.com/package/cypress-downloadfile) or something like that, or it is the same `downloadFile` command that's being defined in this function? I don't understand how it works.

Is there any verification that some file started to be downloaded?